src/terra-dev-site/IssueForm/Helper.jsx

```diff
@@ -0,0 +1,120 @@
+import PropTypes from 'prop-types';
+import Packages from './Packages.json';
+
+// Prop types for all components.
+const propTypes = {
+  /**
+   * List of packages from terra clinical, core, and framework.
+   */
+  packageList: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * Functions that set state with the provided field values.
+   */
+  setContext: PropTypes.func,
+  setDescription: PropTypes.func,
+  setEnvironment: PropTypes.func,
+  setExpected: PropTypes.func,
+  setIssue: PropTypes.func,
+  setMentions: PropTypes.func,
+  setPackage: PropTypes.func,
+  setSolution: PropTypes.func,
+  setSteps: PropTypes.func,
+  setTitle: PropTypes.func,
+  /**
+   * Sets the value of the issue type select box to the selected issue type.
+   */
+  value: PropTypes.string,
+};
+
+// Templates used for preview, errors, and forms.
+const bugTemplate = props => `# Bug Report
+
+## Description
+${props.description}
+
+## Steps to Reproduce
+${props.steps}
+
+## Expected Behavior
+${props.expected}
+
+## Environment
+${props.environment}
+
+${props.context ? `## Additional Context / Screenshots
+${props.context}` : ''}
+
+${props.solution ? `## Possible Solution
+${props.solution}` : ''}
+
+${props.mentions ? `## @ Mentions
+${props.mentions}` : ''}`;
+
+const disclaimerTemplate = `When logging issues to Terra, it is most helpful to have steps to reproduce that just contain the minimal set of components.  Even better would be to be able to reproduce it on <a href="https://engineering.cerner.com/terra-ui" target="_blank">https://engineering.cerner.com/terra-ui</a>. 
+    If there are videos or screenshots that are available, those are helpful as well. 
+
+**Note to Cerner associates:** if you have proprietary information, don't post it directly.  Instead post a link to a site that can be protected by the Cerner IDP (e.g. sharepoint or new relic).`;
+
+const environmentTemplate = `* Component Name and Version: 
```

Comment:
Let's drop the `Component Name` and go with `Component Version` as we should get the component name from the select dropdown.

Reply:
Updated: [d658958](https://github.com/cerner/terra-ui/pull/151/commits/d6589587b241e85489f9f156c29d1dab408f90d0)